client/src/App.js

```diff
@@ -21,8 +23,27 @@ class App extends Component {
           <Route component={NoMatch} />
         </Switch>
       </Router>
+          <Router>
```

Comment:
I don't think we can have 2 separate Routers. Add the login and signup routes to the existing switch.